functions/Get-DbaLinkedServer.ps1

```diff
@@ -0,0 +1,90 @@
+
+.PARAMETER SqlServer
+The SQL Server that you're connecting to.
+
+.PARAMETER Credential
+Credential object used to connect to the SQL Server as a different user
+
+.PARAMETER Databases
+Return information for only specific databases
+
+.NOTES
+Author: Stephen Bennett ( https://sqlnotesfromtheunderground.wordpress.com/ )
+	
+dbatools PowerShell module (https://dbatools.io)
+Copyright (C) 2016 Chrissy LeMaire
+This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+You should have received a copy of the GNU General Public License along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+	
+.LINK
+https://dbatools.io/Get-DbaLinkedServer
+
+.EXAMPLE
+Get-DbaLinkedServer -SqlServer DEV01
+
+Returns all Linked Servers for the SQL instance DEV01
+
+#>
+	[CmdletBinding()]
+	param (
+		[Parameter(Mandatory = $true, ValueFromPipeline = $true)]
+		[Alias("ServerInstance", "SqlServer")]
+		[string[]]$SqlInstance,
+		[System.Management.Automation.PSCredential]$SqlCredential
+	)
+    process
+    {
+        foreach ($Instance in $SqlInstance)
+        {
+            try
+	        {
+	            Write-Verbose "Connecting to $Instance"
+                $server = Connect-SqlServer -SqlServer $Instance -SqlCredential $sqlcredential
+	        }
+	        catch
+	        {
+	            Write-Warning "Failed to connect to: $srv"
+                break
+	        }
+            if($server -contains "\")
+            {
+                $computername = ($server.Name).Split("\")[0]
+                $inst =  ($server.Name).Split("\")[1]
+            }
+            else
+            {
+                $computername = $server.Name
+                $inst =  ""
+            }
+
+            foreach ($ls in $server.LinkedServers)
+            {               
+
+                    $output = [PSCustomObject]@{
+		                ComputerName = $computername
+                        SqlInstance = $inst
+		                LinkedServerName = $ls.Name
+                        RemoteServer = $ls.DataSource
+                        ProductName = $ls.ProductName 
+                        Impersonate = $ls.LinkedServerLogins.Impersonate
+                        RemoteUser = $ls.LinkedServerLogins.remoteuser
+                        Rpc = $ls.Rpc
+                        RpcOut = $ls.RpcOut
+                        LinkedServer = $ls
+                        }
+     
+                    Select-DefaultField -InputObject $output -Property ComputerName, SqlInstance, LinkedServerName, RemoteServer, ProductName, Impersonate, RemoteUser, Rpc, RpcOut
```

Comment:
@SirCaptainMitch that is an internal function...if you are testing it just make sure you have that command in your dbatools\internals folder. It was just added about 6 days ago.